hazelcast/src/main/java/com/hazelcast/internal/metrics/MetricTagger.java

```diff
@@ -19,8 +19,6 @@
 import com.hazelcast.internal.metrics.collectors.MetricsCollector;
 import com.hazelcast.internal.metrics.impl.MetricsRegistryImpl;
 
-import javax.annotation.Nonnull;
-
 /**
  * Interface to be used for tagging a metric.
```

Comment:
Does it make sense to introduce a separate `#withUnitTag(ProbeUnit)` method? I can see lots of duplicate `.withTag("unit", unit.name().toLowerCase())` calls. We could make them less error-prone by introducing that special tag method.